pkg/apis/workflow/v1alpha1/workflow_types.go

```diff
@@ -2267,6 +2316,12 @@ type SuspendTemplate struct {
 	Duration string `json:"duration,omitempty" protobuf:"bytes,1,opt,name=duration"`
 }
 
+// DataTemplate is a template that process and transforms data
+type DataTemplate struct {
```

Comment:
can we put DataTemplate into a new file name `data_template_types.go`?

Reply:
Sure, but I would only do so if we do something similar for the rest of the templates/types that can be grouped

Reply:
Done... will do the grouping in a separate PR